configs/cli/config.toml

```diff
@@ -0,0 +1,26 @@
+[channel]
+  state = ""
+  status = ""
+  topic = ""
+
+[filter]
+  contact = ""
+  email = ""
+  limit = ""
+  metadata = ""
+  name = ""
```

Comment:
`build/mainflux-cli config metadata "info"` for this command getting error 
```error 
error: error converting filter to Uint64 : strconv.ParseUint: parsing "true": invalid syntax


error: no such key.


ok
```

Reply:
@arvindh123  please check once again and see if you get the same error